### PR TITLE
Adding overflow menu for the `BitBreadcrumb` component (#1674)

### LINKED ISSUE #1674: Adding contextual menu functionality for breadcrumb to show list of overflow items.
https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb

In `BitBreadcrumb` by tapping on overflow index item a menu should be opened to show all overflow items.

### PR DESCRIPTION
**Pull request by msangtarash**
this closes #1674

### CODE REVIEW

**Comment by msynk** [comment]
@maryamhdr & @Cyrus-Sushiant 
plz review this PR

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BitBreadcrumb.Fluent.scss [hunk 1]
```diff
@@ -1,22 +1,104 @@
...
+    animation-name: opacity, trans-y;
+    animation-duration: 0.367s;
+    animation-timing-function: cubic-bezier(0.1, 0.9, 0.2, 1);
+    animation-fill-mode: both;
+    background-color: $White;
+    z-index: 1001;
+    overflow: hidden auto;
+    -webkit-font-smoothing: antialiased;
+
+
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BitBreadcrumb.Fluent.scss:69]: Plz, remove blank line

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BitBreadcrumb.Fluent.scss [hunk 2]
```diff
@@ -1,22 +1,104 @@
 ﻿@import "../../Styles/Fluent/colors.fluent.scss", "../../Styles/functions.scss";
 
-.bit-brc-fluent li {
-    display: inline-flex;
-}
+.bit-brc-items-wrapper {
+
+    li {
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BitBreadcrumb.Fluent.scss:5]: Maybe it's better `li` move to `ul` tag

**Review by Cyrus-Sushiant — CHANGES_REQUESTED** [review]

**Comment by msynk** [comment]
@Cyrus-Sushiant plz re-review this PR

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BitBreadcrumb.Fluent.scss [hunk 3]
```diff
@@ -1,22 +1,103 @@
...
 
-.bit-brc-lst-itm-fluent {
-    font-size: rem(20px);
-    font-weight: 600;
-    padding: 0 rem(8px);
-    color: $Grey160;
+.bit-brc-overflow-menu-wrapper {
+    display: none;
+    position: fixed;
+    max-height: 744px;
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BitBreadcrumb.Fluent.scss:54]: Plz, use rem(744px) instead of 744px

**Review by Cyrus-Sushiant — CHANGES_REQUESTED** [review]

**Comment by msynk** [comment]
@maryamhdr review?

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BitBreadcrumb.Fluent.scss (hunk 3 above)
Comment (maryamhdr) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BitBreadcrumb.Fluent.scss:54]: Has this issue been solved? @msangtarash

**Review by maryamhdr — APPROVED** [review]

**Review by maryamhdr — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**